encrypted-media-respec.html

```diff
@@ -839,37 +839,42 @@
                     <p class="note">For example, if <var>audio/video type</var> is Video and the top-level type is not "video" or <var>media types</var> contains non-video codecs.</p>
                   </li>
                   <li><p>If <var>robustness</var> is not the empty string and contains an unrecognized value or a value not supported by <var>implementation</var>, continue to the next iteration. String comparison is case-sensitive.</p></li>
-                  <li><p>If the user agent and <var>implementation</var> do not support playback of encrypted <a def-id="media-data"></a> for the combination of <var>container</var>, <var>media types</var>, <var>robustness</var> and <var>local accumulated configuration</var>, continue to the next iteration.</p>
+                  <li><p>If the user agent and <var>implementation</var> definately support playback of encrypted <a def-id="media-data"></a> for the combination of <var>container</var>, <var>media types</var>, <var>robustness</var> and <var>local accumulated configuration</var>:</p>
                     <p class="note"><var>requested media capability</var> (content type and robustness) must be supported with all previously added requested media capabilities.</p>
-                  </li>
-                  <li>
-                    <p>
-                      Add <var>requested media capability</var> to <var>supported media capabilities</var>.
-                    </p>
-                    <p class="note">
-                      This step ensures that the values of the members of entries in <var>supported media capabilities</var> are exactly the strings supplied in
-                      <var>requested media capability</var> without modification by the User Agent.
-                    </p>
-                  </li>
-                  <li>
-                    <dl class="switch">
-                      <dt>If <var>audio/video type</var> is Video:</dt>
-                      <dd>
+                    <p class="note">If <var>media types</var> is empty (i.e. only container is specified) then
+                        definate support implies support of all possible codecs and parameters for <var>container</var>
```

Comment:
Regarding "codecs and parameters": "codecs" is one of the parameters. RFC6381 (see line 833) defines "codecs" and "profiles".

I think the confusion may come from line 835: "the set of codecs, with parameters, specified by <var>parameters</var>" is confusing. This comes from https://github.com/w3c/encrypted-media/commit/59c28e9a5ab0f53ee6795bf3373f07852f4dd850, but I'm not sure what the intent is. Perhaps we should s/, with parameters,/ and other capabilities/. This would stress codecs while also handling profiles and anything else that might be added to line 833 in the future.